test/remote/gateways/remote_credorax_test.rb

```diff
@@ -79,6 +85,13 @@ def test_successful_purchase_with_google_pay
     assert_equal 'Succeeded', response.message
   end
 
+  def test_successful_purchase_with_network_token
+    response = @gateway.purchase(@amount, @nt_credit_card, @options)
+    assert_success response
+    assert_equal '1', response.params['H9']
+    assert_equal 'Succeeded', response.message
+  end
+
```

Comment:
💬 

Add some unit tests to check the request and evaluate the conditionals.

Reply:
a pair of unit tests added to check the requests in both cases, network token and apple/google pay